### PR TITLE
Demos/CatDog: Make CatDog respond to user programs

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**
This makes CatDog even smarter by having it respond to what programs the user has open, including tailored suggestions.

https://user-images.githubusercontent.com/28656157/155855642-6ea2e53f-a056-4903-8f54-78e65d3b5e8f.mp4

### CODE REVIEW

File: Userland/Demos/CatDog/SpeechBubble.cpp [hunk 1]
```diff
@@ -11,11 +12,21 @@
...
 
-static Array<StringView, 3> messages = {
+static Array<StringView, 3> default_messages = {
     "It looks like you're trying to debug\na program. Would you like some help?"sv,
     "It looks like you're trying to shave\na yak. Would you like some help?"sv,
     "Well Hello Friend!"sv,
 };
+static Array<StringView, 3> pixel_paint_messages = {
+    "It looks like you're creating art.\nWould you like some help?"sv,
+    "It looks like you're making a meme\nfor Discord. :yakthonk:"sv,
```
Comment (drunderscore) [inline Userland/Demos/CatDog/SpeechBubble.cpp:22]: Could we use our custom Unicode Yak emojis here? (https://serenityos.net/~xexxa/10CD)

Comment (kleinesfilmroellchen) [reply Userland/Demos/CatDog/SpeechBubble.cpp:22]: I totally forgot, however maybe it's funnier this way. Let's have others chip in first.

Comment (xslendix) [reply Userland/Demos/CatDog/SpeechBubble.cpp:22]: I think it might look better with our custom fonts

**Comment by kleinesfilmroellchen** [comment]
image

@drunderscore @xslendix  You win!

**Pull request merged**